config/prismjs/dvc-commands.js

```diff
@@ -21,6 +21,7 @@ module.exports = [
   'push',
   'pull',
   'pkg',
+  'plots modify',
   'plots show',
```

Comment:
I think this file uses reverse abc order, so:
```suggestion
  'plots show',
  'plots modify',
```